services/bstats/bstats-players.service.js

```diff
@@ -0,0 +1,61 @@
+'use strict'
+
+const BaseJsonService = require('../base-json')
+
+const Joi = require('joi')
+const schema = Joi.object({
```

Comment:
I think we could use a different schema here that would allow for interfacing with the bStats API directly and avoid the need for the intermediate bridge hop:

```js
const schema = Joi.array()
  .items(Joi.array().items([Joi.number().required(), Joi.number().required()]))
  .required()
```